.travis.yml

```diff
@@ -15,6 +17,7 @@ script:
   - npm test
   - npm run test-ui
   - grunt build
```

Comment:
Do you have to install surge first? `npm i -g surge`??

Reply:
surge is just pushing the code to server it is independent of this project

Reply:
it only push or uplad the files

Reply:
it is working perfectly in community - toolbox